app/editor/src/store/hooks/lookup/useLookup.ts

```diff
@@ -79,153 +91,269 @@ export const useLookup = (): [ILookupState, ILookupController] => {
         store.storeCache(result);
         return result;
       },
+      getLookups: async () => {
+        return await fetchIfNoneMatch<ILookupModel>(
+          'lookups',
+          dispatch,
+          (etag) => lookups.getLookups(etag),
+          (results) => {
+            if (!!results) {
```

Comment:
We don't store the lookup response in local storage.  Instead we store each lookup value in local storage.  This eliminates us having to keep two copies in sync.